common/constants/default.go

```diff
@@ -14,8 +14,8 @@ const (
 	// NodeName is for the clearer log of cloudcore.
 	NodeName = "NodeName"
 
-	KubeEdge          = "kubeedge"
-	KubeEdgeNameSpace = "kubeedge"
+	Name      = "kubeedge"
+	Namespace = Name
```

Comment:
`Namespace` and `Name` don’t sound so clearly when developers import...?

Reply:
like `SystemNamespace` ?

Reply:
For example k8s, defined as `NamespaceDefault`